docker/package/tezos_setup_wizard.py

```diff
@@ -42,7 +42,7 @@
 key_import_modes = {
     "ledger": "From a ledger",
     "secret-key": "Either the unencrypted or password-encrypted secret key for your address",
-    "json": "Faucet JSON file from https://faucet.tzalpha.net/",
+    "json": "Faucet JSON file from https://teztnets.xyz/hangzhounet-faucet/",
```

Comment:
Let's reference https://teztnets.xyz in general without specifying the exact testnet since it's going to change over time

Reply:
Fair note

Reply:
@DMozhevitin I see you changed this in the code but not in the commit message, please update that too